### ISSUE TITLE
totheglory.im no login possible

### ISSUE THREAD

**Issue by manutdd** [Jackett/Jackett#6024]
What is needed in my totheglory cookie in chrome browser and in my pc to tell totheglory that my credentials are valid and legit?

What information was stored in my old windows before I have formatted it and is needed now to be able to login?

![aaaa](https://user-images.githubusercontent.com/55636124/65378181-ca271780-dcbd-11e9-916c-0f49eb8790d8.JPG)

**Comment by garfield69** [comment]
@mouzzampk2014, @drewbitt, @yimingzhaomath: are any of you able to login to totheglory.im or has the site been officially censored by cn?

**Comment by yimingzhaomath** [comment]
I can't log on. It is a temporary thing. Right now, the site is
unreachable, but the tracker still works. Website login is currently
restricted to website admins. But I expect it will reopen in time. (This
situation has happened before in April.)

**Comment by mouzzampk2014** [comment]
I am sorry for late reply. Yes site is down

![1](https://user-images.githubusercontent.com/44548525/65378518-5bdc5800-dcb1-11e9-8465-3785f3f8b07b.jpg)

**Comment by manutdd** [comment]
Let me try again!
The site is working normally, torrents are seeding BUT the login is blocked for "new" users.
If you did not cleared your cookie or reinstalled windows enerything is ok because google chrome or firefox remembers or has your cookie BUT in my case I reinstalled my windows and now I cannot login because I do not have that cookie.
If i have saved my cookie BEFORE I had reinstalled my windows now I could import it in chrome and login successful BUT I do not have that cookie, so my question is this:
Is anyone here kind enough to just share the TTG cookie so that I can import it and login to TTG?
It is not something personal and private just the cookie that is using anyone who has an account on TTG.

**Comment by mouzzampk2014** [comment]
I just reinstalled my Ubuntu yesterday. Is there any other way I can get the cookie for you?

**Comment by mouzzampk2014** [comment]
Cam I do that from within jackett?

**Comment by manutdd** [comment]
I am not a cookie expert but this is how my cookie looks like now when I access TTG:
https://imgur.com/a/vIDEoPB

**Comment by manutdd** [comment]
Also I would need some help from someone who has some knowledge on how cookie works.
I have this information from someone:

"Install this addon https://addons.mozilla.org/en-US/fir...quick-manager/, you can find similar ones on other browsers. Open the addon, navigate to the website, you see something like this https://imgur.com/a/sHKYQhx. Here the first one is your userid and the second is your password encoded by md5. If the tracker save these cookies then it is pretty simple to add them back. You know your ID, your password can be easily encoded using any online websites."

my response was this:

Ok so let's take chrome because I have used it all my life, moziila not.
I have EditThisCookie addon and there I have the export options and I get something like this:

[
{
"domain": ".totheglory.im",
"expirationDate": some_random_numbers,
"hostOnly": false,
"httpOnly": true,
"name": "__cfduid",
"path": "/",
"sameSite": "unspecified",
"secure": true,
"session": false,
"storeId": "0",
"value": "some_random_numbers_and_letters",
"id": 1
},
{
"domain": "totheglory.im",
"hostOnly": true,
"httpOnly": false,
"name": "PHPSESSID",
"path": "/",
"sameSite": "unspecified",
"secure": false,
"session": true,
"storeId": "0",
"value": "some_random_numbers_and_letters",
"id": 2
}
]

where I have replaced the numbers and letters with "some_random_numbers" because I don't know if this is private information because again I dont't know how cookies work and I would like to learn.
So my username is "value": xxxxxxxxxxxxxxxxxxx? above "id": 1
and my password is "value": xxxxxxxxxxxxxxxxxxxx? above "id": 2
AND these numbers and letters are generated with this md5 generator:

https://md5calc.com/hash/md5/my+username

https://md5calc.com/hash/md5/my+password

And after I replace both "value" fields I just IMPORT to TTG site using EditThisCookie??

I'm I correct? it's that simple? I'm I wrong and far away for getting this trick to work?


and now I'm stuck on that cookie!

**Comment by garfield69** [comment]
@mouzzampk2014  ye,s you should be able to find the cookie data in the totheglory indexer profile file.
take a look at the  ~/.config/Jackett/Indexers folder and you should find the totheglory.json file.
edit it with a plain text editor, and look for the cookieheader id, should look something like this
```json
  {
    "id": "cookieheader",
    "type": "hiddendata",
    "name": "CookieHeader",
    "value": "**redacted**"
  },
```
but be aware the value may contain personal login details such as username/password/userid etc so you will want to redact those parts out before publishing the cookieheader here ;-)

**Comment by ngosang** [comment]
is this fixed?

**Comment by garfield69** [comment]
there is no way that anyone can create valid cookies by modifying some one else's cookies in order to provide a valid cookie string tailored for the user's credentials.
no-one but the site knows how they generate their cookie values, suing whatever encryption method, hashing method, what ever.
this user cannot be helped with his issue here.
